apps/playwright/playwright.config.ts

```diff
@@ -0,0 +1,86 @@
+import { defineConfig, devices } from "@playwright/test";
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
+  testDir: "./tests",
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
```

Comment:
You don't anticipate any trouble running this in a GH workflow, right?

Reply:
Yeah, so my intention was to do the minimum amount of work to see what people thought of the style of writing tests before exploring it further.

 It seems like the ones who have reviewed it are tentatively positive about it so we may be able to start filing in some gaps like the CI thing. Do you think it is worth exploring that now? This is obviously a requirement and they have some pretty good [documentation around it ](https://playwright.dev/docs/ci-intro)(one of the things I enjoyed about working with it).

Reply:
No, that's fine.  It was a reasonable scope limit.  As long as it's well documented, that's enough for me right now.